components/director/internal/tenantfetchersvc/subscriber_test.go

```diff
@@ -440,7 +440,7 @@ func TestSubscribeRegionalTenant(t *testing.T) {
 			LabelServiceFn: func() *automock.LabelService {
 				labelSvc := &automock.LabelService{}
 				labelSvc.On("GetLabel", context.TODO(), tenantID, &getLabelInput).Return(&testLabel, nil).Once()
-				labelSvc.On("UpdateLabel", context.TODO(), tenantID, testLabel.ID, &updateLabelInput).Return(testError).Once()
+				labelSvc.On("UpdateLabel", context.TODO(), tenantID, testLabel.ID, &updateLabelInput).Return(testError).Twice()
```

Comment:
Write new test cases in TestSubscribeRegionalTenant and TestUnSubscribeRegionalTenant when UpdateLabel succeeds after one retry(On first UpdateLabel return testError, on second UpdateLabel return nil).